ADR/0039-spdx-support.md

```diff
@@ -0,0 +1,144 @@
+# 0. Record architecture decisions
+
+Date: 2024-09-24
+
+## Status
+
+Proposed
+
+## Context
+
+SPDX SBOM format enables additional features not available in cyclondedx like multiple purl attributes per component. SPDX is also a widely adopted standard for software bill of materials.
+This ADR describes how to enable use of SPDX SBOM format in Konflux.
+
+## Decision
+
+
+### SBOM lifecycle in build pipeline
+
+At the start SBOMs are generated by cachi2 and syft. At later phase of the build pipeline, SBOMs from base images are extracted and merged toghether with SBOMs generated for the currently build container. When switching to spdx, newly built container image can be used as base image another container image. Therefore any tooling/task which works with sboms has to be able to work with both formats.
+Due to that fact, all tools and tasks should implement sbomType attribute which will be used to determine sbom format expected on the input and produced on the output. It will also allow tools to be tested for spdx before whole ppile is switched to spdx.
```

Comment:
```suggestion
Due to that fact, all tools and tasks should implement the sbomType attribute which will be used to determine the sbom format expected on the input and produced on the output. It will also allow tools to be tested for spdx before whole pipeline is switched to spdx.
```